OpenRA.Mods.Common/Scripting/Global/TriggerGlobal.cs

```diff
@@ -211,8 +211,27 @@ public void OnAllRemovedFromWorld(Actor[] actors, LuaFunction func)
 						return;
 
 					if (!group.Any())
-						using (f)
-							f.Call().Dispose();
+					{
+						// Do not dispose the 'original' function f, we may need it again
```

Comment:
Please change this to `// Functions can only be .Call()ed once, so operate on a copy so we can reuse it later`